pkg/pipelines/pipelines.go

```diff
@@ -168,15 +162,15 @@ func validatePipelineRunCancel(c *clients.Clients, prname, namespace string) {
 
 	log.Printf("Waiting for PipelineRun %s in namespace %s to be cancelled", prname, namespace)
 	if err := wait.WaitForPipelineRunState(c, prname, wait.FailedWithReason("PipelineRunCancelled", prname), "PipelineRunCancelled"); err != nil {
-		testsuit.T.Errorf(fmt.Sprintf("Error waiting for PipelineRun `pear` to finished: %s", err))
+		testsuit.T.Errorf(fmt.Sprintf("Error waiting for PipelineRun `%s` to finished: %s", prname, err))
```

Comment:
finished or cancelled?

Reply:
it's `finished` only but with failed reason as `pipelineRunCancelled`